recipes/hat-phasing/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "HAT-phasing" %}
+{% set version = "0.1.0" %}
+{% set d_name = "HAT_phasing" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ d_name }}-{{ version }}.tar.gz"
+  sha256: e88937a73bb43dbba26d40c09f88d0f095880a577ddd82cf31cee5cf833b1f2d
+
+build:
+  number: 0
+  noarch: generic
+  entry_points:
+    - HAT = hat.HAT_main:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - biopython ==1.79
```

Comment:
you only need python and pip in the host section

Reply:
Thanks for the review. I removed the rest